File: API/Repositories/Models/Customers.cs
namespace API.Repositories.Models
{
    public class Customers
    {
        private int Id { get; set; }
        public string Login { get; set; }
        public string Password { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set;}
        public string AddressLine1 { get; set; }
        public string AddressLine2 { get; set; }
        public string PostCode { get; set; }
    }
}

File: API/ClientApp/src/app/models/Customers.ts
export class Customers {

    constructor(
        
    ) {
        this.id = 0;
        this.firstName = "";
        this.lastName = "";
        this.login = "";
        this.password = "";
        this.birthdate = new Date();
        this.addressLine1 = "";
        this.addressLine2 = "";
        this.postCode = "";
    }

    public id: number;
    public login: string;
    public password: string;
    public firstName: string;
    public lastName: string;
    public addressLine1: string;
    public addressLine2: string;
    public postCode: string;
    public birthdate: any;
}